src/app/shared/issue/duplicateOf/duplicate-of.component.html

```diff
@@ -9,9 +9,11 @@
   <mat-select [style.display]="isEditing ? 'block' : 'none'" class="no-arrow" placeholder="-" [value]="issue.duplicateOf"
               (selectionChange)="updateDuplicateStatus($event)" (openedChange)="handleSelectionOpenChange($event)">
     <mat-select-trigger></mat-select-trigger>
-    <mat-option *ngFor="let issue of duplicatedIssueList | async" [disabled]="dupIssueOptionIsDisabled(issue)" [value]="issue.id">
-      <span class="mat-body-strong"> #{{issue.id}}: </span>  <span class="mat-body">{{issue.title}}</span>
-      <span *ngIf="dupIssueOptionIsDisabled(issue)" class="mat-caption" style="color: #f44336"> ({{getDisabledDupOptionErrorText(issue)}}) </span>
+    <mat-option [matTooltip]="issue.title" [matTooltipDisabled]="disableTooltip(issue)" [matTooltipPosition]="'left'" *ngFor="let issue of duplicatedIssueList | async" [disabled]="dupIssueOptionIsDisabled(issue)" [value]="issue.id">
+      <div id="duplicate-dropdown-issuedetails" [ngStyle]="{'width': dupIssueOptionIsDisabled(issue) ? '170px' : 'auto'}">
```

Comment:
Ids should be unique. Over here, you are iterating over the same id. You can define it as a class instead of id!

Reply:
Ah! I understand, I had the impression that because this was the only div under the duplicates drop down I should use ID. I'll change it.